### ISSUE TITLE
[Adaptive Cards> ShowCardWizard.json]: Screen Reader is not announcing the state of the Name, Address and Phone/Email button. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8976]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to 'ShowCardWizard.Json' button by using Tab key and activate it.
4. ShowCardWizard.Json adaptive card will appear on right sight of the pane.
5. Navigate to ShowCardWizard.Json card by using Tab key.
6. Press the tab key and navigate to the 'Name, Address and Phone/Email' button and observe the issue.

**Actual Result:**
While navigating to the 'Name, Address and Phone' button, screen reader is not conveying the state information as 'Expand/Collapsed'. It is just announcing as **'Name button.'**

**Expected Result**
Screen Reader should announce the state for the button control. When the focus lands on the Name button then it should announce the information as 'Name button collapsed' after pressing enter key it should announce as 'Expended.'

**Note:**
Same issue repro throughout for all the this button control.

**User Impacted:**
User depending on screen reader might get affected if screen reader does not pass State information about the control. Screen reader user will not able to know the property of that control and will face problem while interacting with it.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/6d1d546e-7678-4bf8-8f14-2442ca3fd863

![Screen Reader is not announcing the state of the Name, Address and Phone Email button](https://github.com/user-attachments/assets/cc38c18f-3478-46a8-a0f3-f9d093403969)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
